gapic/src/main/com/google/gapid/views/TextureList.java

```diff
@@ -0,0 +1,551 @@
+/*
```

Comment:
It'd be nice if git tracked this as a copy of `TextureView.java`.

Reply:
I agree, but unfortunately [git doesn't support](https://stackoverflow.com/questions/1043388/record-file-copy-operation-with-git) tracking of copies and known workarounds don't work anymore with recent git versions.